src/imagepicker.ios.ts

```diff
@@ -28,7 +28,15 @@ export class ImagePicker extends data_observable.Observable {
     }
 
     get hostController() {
-        return this.hostView ? this.hostView.viewController : UIApplication.sharedApplication.keyWindow.rootViewController;
+        let vc = this.hostView ? this.hostView.viewController : UIApplication.sharedApplication.keyWindow.rootViewController;
+        while (
+            page.presentedViewController
```

Comment:
page is not defined

Reply:
fixed foolish typo 🤦‍♂️ 
Thank you for the heads up. and happy new year 🍾